Microsoft.DotNet.Interactive/IExtensibleKernel.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using MLS.Agent.Tools;
+
+namespace Microsoft.DotNet.Interactive
+{
+    public interface IExtensibleKernel
+    {
+        RelativeDirectoryPath ExtensionsPath { get; }
```

Comment:
An interface should expose behavior, not state. 

A specific thing to consider about exposing a path as state is that a path is just a means to get hold of an assembly or script,  so it's more relevant to the loader, not the kernel. And keep in mind there's more than one path that might be valid for a given kernel (e.g. an assembly path and a script path both for C# extensions.)

Reply:
Now the IExtensibleKernel defines a method that can load extensions inside a given directory